gdal/ogr/ogr_srs_esri.cpp

```diff
@@ -1884,17 +1884,20 @@ OGRErr OGRSpatialReference::morphToESRI()
           }
         }
 
+        // pszProjection might be deleted in DeleteParamBasedOnPrjName,
+        // so make and use a copy.
+        const string osProjection(pszProjection);
```

Comment:
std::string ?

Reply:
Yup.  I've still got one more round of fixes to do.